buildstockbatch/test/test_inputs/enforce-validate-measures-bad-2.yml

```diff
@@ -11,23 +11,46 @@ sampler:
     n_datapoints: 30
 
 workflow_generator:
-  type: residential_default
+  type: residential_hpxml
   args:
-    residential_simulation_controls:
-      timesteps_per_hr: 4
-      begin_month: 1
-      begin_day_of_month: 1.5 # intentional bad input data type for unit testing input validation
-      end_month: 12
-      end_day_of_month: 31
-    timeseries_csv_export:
-      reporting_frequency: Huorly # intentionally misspelled for unit testing input validation
-    #  include_enduse_subcategories: true
-      output_variable:
-        - Zone Mean Air Temperature
-    simulation_output:
-      include_enduse_subcategory: true
+    build_existing_model:
+      simulation_control_timestep: 60
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1.5 # intentional bad input data type for unit testing input validation
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_MidCase_15
+        type: CO2e
+        elec_folder: data/cambium/LRMER_MidCase_15
+
+    utility_bills:
+      - scenario_name: Bills
+
+    simulation_output_report:
+      timeseries_frequency: huorly # intentionally misspelled for unit testing input validation
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_emission_fuels: true
+      include_timeseries_emission_end_uses: true
+      include_timeseries_hot_water_uses: true
+      include_timeseries_total_loads: true
+      include_timeseries_component_loads: true
+      include_timeseries_unmet_hours: true
+      include_timeseries_zone_temperatures: true
+      include_timeseries_airflows: true
+      include_timeseries_weather: true
```

Comment:
Seems like you could probably (based on what you're testing) trim a lot of this as it may have no bearing on the actual test... (same comment for test ymls below)